CMakeLists.txt

```diff
@@ -118,6 +118,8 @@ endif()
 # QuickJS library
 #
 
+xoption(BUILD_QJS_LIBC "Build quickjs-libc.c" OFF)
```

Comment:
Please add a slightly more descriptive message: "Build standard library modules as part of the library" or something like that...